code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -46,42 +57,47 @@
 
 /mob/living/simple_animal/hostile/asteroid/fugu/Aggro()
 	..()
-	Inflate()
+	E.Activate()
 
-/mob/living/simple_animal/hostile/asteroid/fugu/verb/Inflate()
-	set name = "Inflate"
-	set category = "Fugu"
-	set desc = "Temporarily increases your size, and makes you significantly more dangerous and tough."
-	if(wumbo)
-		to_chat(src, "<span class='notice'>You're already inflated.</span>")
+/datum/action/innate/fugu
+	icon_icon = 'icons/mob/actions/actions_animal.dmi'
+
+/datum/action/innate/fugu/expand
+	name = "Inflate"
+	desc = "Temporarily increases your size, and makes you significantly more dangerous and tough! Do not bully the fugu!"
+	button_icon_state = "expand"
+
+/datum/action/innate/fugu/expand/Activate()
+	var/mob/living/simple_animal/hostile/asteroid/fugu/F = owner
+	if(F.wumbo)
+		to_chat(F, "<span class='notice'>YOU'RE ALREADY WUMBO!</span>")
 		return
-	if(inflate_cooldown)
-		to_chat(src, "<span class='notice'>We need time to gather our strength.</span>")
+	if(F.inflate_cooldown)
+		to_chat(F, "<span class='notice'>You need time to gather our strength.</span>")
 		return
```

Comment:
You need time to gather **your** strength

Reply:
ty